### DISCUSSION TITLE
"Search Results" pane should be part of the script editor when it is floating

### DISCUSSION THREAD

**Discussion by kemgoblin** [godotengine/godot-proposals#8356] [Editor]
I like to use the script editor in a separate window (floating) but when I use Find in Files the Search Results pane opens up in the editor window, so I have to keep going back and forth between the two.

It would be nice if this pane (and maybe some of the others) were also accessible in the script editor when it is floating.

Quick mockup:
![image](https://github.com/godotengine/godot-proposals/assets/111526442/cd13adc1-3f98-44d3-a93a-a6e7cdca073e)

**Comment by Calinou** [comment]
The bottom panel isn't designed to be detached so this is really difficult to implement.
